Week-9/todoapplication/src/components/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from "react";
+import { Todo, TodoContext } from "./";
+import "../AppWideCSS.css";
+
+function TodoList() {
+  const importedTodoContext = useContext(TodoContext);
+
+  const mapCallbackToRender = (todo) => {
+    return <Todo todo={todo} key={todo.id} />;
+  };
+
+  const renderTodos = importedTodoContext.todoList.map(mapCallbackToRender);
```

Comment:
importedTodoContext.todoList.map(mapCallbackToRender);  2/10

=> 

importedTodoContext.todoList.map(renderTodoItem); 10/10

renderTodoItem = (todo) => {
    return <Todo todo={todo} key={todo.id} />;
}

Reply:
renderTodos  => TodoElementList

Reply:
Your naming is perfect.
Changed names on the repo...